hbase-client/src/test/java/org/apache/hadoop/hbase/client/TestTableDescriptorBuilder.java

```diff
@@ -292,6 +347,13 @@ public void testStringCustomizedValues() {
       "'testStringCustomizedValues', " +
         "{TABLE_ATTRIBUTES => {DURABILITY => 'ASYNC_WAL'}}, {NAME => 'cf', BLOCKSIZE => '1000'}",
       htd.toStringCustomizedValues());
+
+    htd = TableDescriptorBuilder.newBuilder(htd).setMaxFileSize("10737942528").build();
+    assertEquals(
+      "'testStringCustomizedValues', " +
+        "{TABLE_ATTRIBUTES => {DURABILITY => 'ASYNC_WAL', "
+        + "MAX_FILESIZE => '10737942528 B (10 GB 512 KB)'}}, {NAME => 'cf', BLOCKSIZE => '1000'}",
```

Comment:
Does it have to have a space between the number and the unit? Can it be 10GB instead of '10 GB'?

Reply:
I copied the pretty print format of TTL. But I‘m with you, the space is not necessary. Will update.